sycl/include/sycl/handler.hpp

```diff
@@ -161,6 +160,12 @@ template <class _name, class _dataT, int32_t _min_capacity, class _propertiesT,
 class pipe;
 }
 
+namespace ext ::oneapi ::experimental {
+// State of a queue with regards to graph recording,
+// returned by info::queue::state
+struct image_descriptor;
```

Comment:
This comment seems incorrectly applied to the `image_descriptor`. Did it not originally belong to the `queue_state` declaration?

```
// State of a queue with regards to graph recording,
// returned by info::queue::state
enum class queue_state { executing, recording };
```

Reply:
That's right, thank you.